frontend/public/components/console-app.jsx

```diff
@@ -0,0 +1,116 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as UIActions from '../actions/ui';
+import CloudShell from '@console/app/src/components/cloud-shell/CloudShell';
+import store from '../redux';
```

Comment:
Nit: please try to keep the import order convention (vendor first, then `@console/xxx`, then relative paths).